tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php

```diff
@@ -104,6 +140,17 @@ public function testNodeFromRegistry()
         $this->subscriber->handleNodeFromRegistry($this->persistEvent->reveal());
     }
 
+    public function testReorderNodeFomRegistry()
```

Comment:
I introduced this policy when writing this library. It does add value,
otherwise I would not have added them. Please indulge me.

On Tue, Jul 21, 2015 at 11:42:50PM -0700, Daniel Rotter wrote:
>    In [1]tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php:
> 
>  > @@ -104,6 +140,17 @@ public function testNodeFromRegistry()
>  >          $this->subscriber->handleNodeFromRegistry($this->persistEvent->reveal());
>  >      }
>  >
>  > +    public function testReorderNodeFomRegistry()
> 
>    This comment also doesn't add any value. This is a policy which leads to
>    comments like the ones above. I'll add it now, but I want this be
>    discussed now, since you have introduced this "policy" without any
>    discussion in the team. /cc [2]@chirimoya [3]@wachterjohannes
> 
>    As already said I have a VERY strong opinion against that.
> 
>    —
>    Reply to this email directly or [4]view it on GitHub.
> 
>    Reverse link: [5]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu-document-manager/pull/28#discussion_r35185380
>    2. https://github.com/chirimoya
>    3. https://github.com/wachterjohannes
>    4. https://github.com/sulu-io/sulu-document-manager/pull/28/files#r35185380
>    5. https://github.com/sulu-io/sulu-document-manager/pull/28/files#r35185380